src/RA_StringUtils.h

```diff
@@ -174,7 +174,7 @@ class StringBuilder
     void Append(const std::string& arg)
     {
         PendingString pPending;
-        pPending.Ref.String = &arg;
+        pPending.Ref = &arg;
         pPending.DataType = PendingString::Type::StringRef;
         m_vPending.emplace_back(std::move(pPending));
```

Comment:
These `emplace_back`s should actually be `push_back `since the purpose of `emplace` is to construct in-place via perfect forwarding (no copying or moving). But ignored them for later.